File: Prince_Tiwari_Assignment1/Rhombus_Pattern.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Prince_Tiwari_Assignment1
{
    internal class Rhombus_Pattern
    {
        public static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            for (int i = 1; i <= n; i++)
            {
                for (int j = 1; j <= n - i; j++)
                {
                    Console.Write(" ");
                }
                for (int k = 1; k <= n; k++)
                {
                    Console.Write("*");
                }
                Console.WriteLine();
            }
        }
    }
}

File: Prince_Tiwari_Assignment1_VB/Rhombus_Pattern.vb
Public Class Rhombus_Pattern
    Public Shared Sub Main(args As String())
        Dim n As Integer = Convert.ToInt32(Console.ReadLine())

        For i As Integer = 1 To n
            For j As Integer = 1 To n - i
                Console.Write(" ")
            Next j
            For k As Integer = 1 To n
                Console.Write("*")
            Next k
            Console.WriteLine()
        Next i
    End Sub
End Class
